tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1042,7 +1041,7 @@ def test_single_sign_on_ldap_ad_server(
     # register the satellite with AD for single sign-on and update external auth
     try:
         # enable the foreman-ipa-authentication feature
-        run_command(cmd='satellite-installer --foreman-ipa-authentication=true', timeout=800)
+        run_command(cmd='satellite-installer --foreman-ipa-authentication=true', timeout=800000)
```

Comment:
I really don't think we should this much big timeout I have seen tests are getting hanged.